### DISCUSSION TITLE
Create a chart out of all results

### DISCUSSION THREAD

**Discussion by mkbctrl** [appbaseio/reactivesearch#1638] [Q&A]
Thanks for creating this awesome library. 

I have a question, I struggle to find an answer for.

I have a following list of items:
<img width="1421" alt="Screenshot 2021-03-28 at 16 32 00" src="https://user-images.githubusercontent.com/10107362/112755952-48672700-8fe3-11eb-912d-d383affdc5d6.png">

this list shows 30 rows, and is paginated if the result list is longer than that.

My issue is that I want to create a chart that will present some of those informations as a summary, in example total count of all stream users. In order to do so, I need the data for all the queried records, even those hidden behind the pagination, and I can't access them.

Do you think I approach this issue correctly, or would you approach it differently?

**Comment by siddharthlatest** [comment]
@mkbctrl You would necessarily need to fetch the summary (as you say) in its entirety to create this chart. However, this can potentially be done as a number of aggregations v/s getting all the hits.

I would then recommend using something like [`<ReactiveComponent/>`](https://docs.appbase.io/docs/reactivesearch/v3/advanced/reactivecomponent/) to have full control over the query and rendering parts.
